features/ouders.feature

```diff
@@ -21,14 +21,12 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
     | fields              | ouders.ouderAanduiding,ouders.naam.voornamen |
     Dan heeft de ouder met ouderAanduiding '1' de volgende gegevens
```

Comment:
Is dit niet eigenlijk een heel vreemde formulering? Wat in regel 22 wordt beweerd is immers altijd waar. Hetzelfde geldt voor regel 28.

Reply:
Dit ziet er wat dubbelop uit omdat dit veld in het LO dezelfde naam heeft als de property in de GBA_versie. Het klopt echter wel.